torch/_dynamo/variables/dicts.py

```diff
@@ -355,6 +355,13 @@ def call_method(
     def unpack_var_sequence(self, tx):
         return [x.vt for x in self.items.keys()]
 
+    def call_hasattr(self, tx, name):
```

Comment:
Is this already generated to any class using `__slots__`? If not, that would be neat.

Reply:
Say more. We can make it more generic by sharing this across ListVariable, ConstDictVariable and just using `__slots__`. But, I think we will still have to go tracker by tracker.

Reply:
If '__slots__' is defined on a class, instances of the created objects won't have a '__dict__' I'm pretty sure. Therefore, they shouldn't allow values that are not in the '__slots__' unless they have a dynamic '__setattr__' or have a base class which adds a '__dict__'. Trying to see if there is an easy Python utility here to detect this, but I'm coming up blank except from some very esoteric CPython C calls.